installation/alpine.sh

```diff
@@ -0,0 +1,591 @@
+#/bin/ash
+
+# All-in-one installation without Docker
+
+immich_ver=1.27.0_37-dev
```

Comment:
So this one is hardcoded; I assume we must change this per release?

Reply:
@Chiogros FYI, you can use `https://api.github.com/repos/immich-app/immich/releases/latest` to fetch the latest version.

Reply:
Yes, this one must be changed for each new release.

Thanks @panoti for your tip. 
However, looking at the latest release available could break the script when Alpine will be updated: in the future, if someone wants to install Immich on an old Alpine version (e.g. 3.14), using this link will lead the script to download an Alpine 3.15,16... Immich compatible version, which will not work.

Reply:
In top of that @alextran1502, if Dockerfiles are updated, we need to review the script to stick with the modified Dockerfile (install new packages mainly).
We don't need to change anything else while the mobile/server/web/ML code is updated.